app/pdfs/distribution_pdf.rb

```diff
@@ -1,5 +1,7 @@
 class DistributionPdf
   include Prawn::View
+  include ItemsHelper
+  include ActionView::Helpers::NumberHelper
```

Comment:
It looks like the only place you're using this helper is in `ItemsHelper` -- could you please:

 * remove this include from this file (unless you are in fact using it here, in which case I'm missing something!)
 * Reference it in the `ItemsHelper` like: `ActionController::Base.helpers.number_to_currency` 

This will help ensure that we're not smuggling in extra code that's not being used. 

Thanks!